spectec/src/backend-latex/config.ml

```diff
@@ -23,26 +12,12 @@ type config =
 
 type t = config
 
-
 let default =
-  { anchors = [];
-    macros_for_ids = false;
+  { macros_for_ids = false;
     macros_for_vdash = false;
     include_grammar_desc = false;
   }
 
-let latex =
-  { default with
-    anchors = [
-      {token = "@@"; prefix = "$"; suffix ="$"; indent = ""};
-      {token = "@@@"; prefix = "$$\n"; suffix = "\n$$"; indent = ""};
-    ]
-  }
+let latex = default
 
-let sphinx =
-  { default with
-    anchors = [
-      {token = "$"; prefix = ":math:`"; suffix ="`"; indent = ""};
-      {token = "$$"; prefix = ".. math::\n   "; suffix = ""; indent = "   "};
-    ]
-  }
+let sphinx = default
```

Comment:
The latex/sphinx shorthands are no longer meaningful in this module, we can remove them.